apps/main/src/lend/layout/index.tsx

```diff
@@ -1,79 +1,5 @@
-import { networksIdMapper } from '../networks'
+import { ReactNode } from 'react'
 
-const BaseLayout = ({ children }: { children: ReactNode }) => {
-  const globalAlertRef = useRef<HTMLDivElement>(null)
-  const params = useParams() as UrlParams
-  const { network: networkId } = params
-  const layoutHeight = useLayoutStore((state) => state.height)
-  const setLayoutHeight = useLayoutStore((state) => state.setLayoutHeight)
-  const bannerHeight = useLayoutStore((state) => state.height.globalAlert)
-  const minHeight = useMemo(() => layoutHeightKeys.reduce((total, key) => total + layoutHeight[key], 0), [layoutHeight])
-
-  useLayoutHeight(globalAlertRef, 'globalAlert', setLayoutHeight)
-
-  const sections = useMemo(() => getSections(params), [params])
-  const chainId = networksIdMapper[networkId]
-
-  return (
-    <Container globalAlertHeight={layoutHeight?.globalAlert}>
-      <Header chainId={chainId} sections={sections} globalAlertRef={globalAlertRef} networkId={networkId} />
-      <Main minHeight={minHeight}>{children}</Main>
-      <Footer appName="lend" networkId={networkId} headerHeight={useHeaderHeight(bannerHeight)} />
-    </Container>
-  )
-}
-
-const getSections = ({ network }: UrlParams): NavigationSection[] => [
-  {
-    title: t`Documentation`,
-    links: [
-      { href: 'https://news.curve.finance/', label: t`News` },
-      { href: 'https://resources.curve.finance/lending/understanding-lending/', label: t`User Resources` },
-      { href: 'https://docs.curve.finance', label: t`Developer Resources` },
-      { href: getPath({ network }, `${ROUTE.PAGE_DISCLAIMER}?tab=lend`), label: t`Risk Disclaimers` },
-      { href: getPath({ network }, ROUTE.PAGE_INTEGRATIONS), label: t`Integrations` },
-      { href: 'https://resources.curve.finance/glossary-branding/branding/', label: t`Branding` },
-      ...(isChinese() ? [{ href: 'https://www.curve.wiki/', label: t`Wiki` }] : []),
-    ],
-  },
-  {
-    title: t`Security`, // audits, bug bounty, dune analytics, curve monitor & crvhub
-    links: [
-      { href: 'https://docs.curve.finance/references/audits/', label: t`Audits` },
-      { href: 'https://docs.curve.finance/security/security/', label: t`Bug Bounty` },
-      { href: 'https://dune.com/mrblock_buidl/Curve.fi', label: t`Dune Analytics` },
-      { href: 'https://curvemonitor.com', label: t`Curve Monitor` },
-      { href: 'https://crvhub.com/', label: t`Crvhub` },
-    ],
-  },
-]
-
-const Main = styled.main<{ minHeight: number }>`
-  margin: 0 auto;
-  max-width: var(--width);
-  min-height: ${({ minHeight }) => `calc(100vh - ${minHeight}px - var(--header-height))`};
-  width: 100%;
-`
-
-const Container = styled.div<{ globalAlertHeight: number }>`
-  display: flex;
-  flex-direction: column;
-  position: relative;
-  width: 100%;
-  min-height: ${({ globalAlertHeight }) => `calc(100vh - ${globalAlertHeight}px)`};
-`
+const BaseLayout = ({ children }: { children: ReactNode }) => children
```

Comment:
I'd argue we could get rid of these `index.tsx` files in the `layout` folder of each app, and move the contents to the `client.tsx` file of the corrsponding app in the `app` folder. One less confusing redirection